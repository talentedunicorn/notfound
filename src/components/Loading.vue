<template>
  <div>
    <h1>loading...</h1>
  </div>
</template>

<script>
  export default {
    name: 'Loading'
  }
</script>

<style scoped>
  div {
    z-index: 10;
    position: fixed;
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
    background: var(--gray);
    display: grid;
    align-items: center;
    justify-content: center;
  }

  h1 {
    margin: 0;
  }
</style>
