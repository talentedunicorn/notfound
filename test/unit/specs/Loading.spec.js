import { shallowMount } from '@vue/test-utils'
import Loading from '@/components/Loading'

describe('Loading.vue', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(Loading)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
