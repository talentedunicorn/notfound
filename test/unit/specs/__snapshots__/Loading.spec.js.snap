// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading.vue should match snapshot 1`] = `
<div>
  <h1>loading...</h1>
</div>
`;
